packages/common/src/utils/challenges.ts

```diff
@@ -537,6 +535,10 @@ export const getChallengeStatusLabel = (
     case ChallengeName.TrackUpload:
       return `${challenge.current_step_count ?? 0}/3 Uploaded`
 
+    // Special-case as this is an infinite aggregate challenge (will always be in-progress)
```

Comment:
i still feel like it's off a bit seeing the `new` after claiming once already 🤔

Reply:
fair point